app/src/views/items/EditItem.vue

```diff
@@ -81,9 +84,10 @@ import * as CodeMirror from "codemirror";
 import { linter } from "@codemirror/lint";
 import { oneDark } from "@codemirror/theme-one-dark";
 import { json, jsonParseLinter } from "@codemirror/lang-json";
+import parseJson from "parse-json";
```

Comment:
The result from `parseJson` & catch block of `JSON.parse`  looks similar. Should we use this package? Changes in https://github.com/kritish-dhaubanjar/dynamodb-dashboard/pull/1 looks simpler.

Reply:
What kind of error messages did you try?